ArkEcosystem/laravel-foundation
# feat: require password to remove account

## Summary

https://app.clickup.com/t/1md1ht3

Updates the user delete form to require the user password.

### To test:
Go to marketsquare and add this PR dependency, try to delete your own account it should ask and validate your password.

![image](https://user-images.githubusercontent.com/17262776/137032320-4d817571-be32-4258-a651-165ef2b2c126.png)


## Checklist
